### PR TITLE
Add Qingg v2.3.7

### PR DESCRIPTION
**Pull request by Igotit**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

The first submission of Qingg Input Method to Homebrew Cask.

### CODE REVIEW

File: Casks/qingg.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask 'qingg' do
+  version '2.3.7'
+  sha256 'f10021d5d650f0bfb4ef1e7448bd0faae8c2b93adb31346db947135c4ffa95fb'
+
+  url "https://qingg.im/download/Qingg-#{version}.dmg"
+  appcast 'https://qingg.im/sparkle/appcast.php',
+          checkpoint: 'fca8f7ae6e07c7cea158d0e37ff34b44931287405df78aedfcf32aa9aebd11f6'
+  name 'Qingg'
```
Comment (vitorgalvao) [inline Casks/qingg.rb]: Too many.

Keep just

```ruby
name 'QinggIM'
name '清歌输入法'
```

File: Casks/qingg.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  homepage 'https://qingg.im/mac/'
+  license :gratis
+
+  auto_updates true
+
+  pkg 'Qingg.pkg'
+
+  uninstall pkgutil: 'com.aodaren.Qingg.pkg'
+
+  zap delete: ['~/Library/Application Support/Qingg',
```
Comment (vitorgalvao) [inline Casks/qingg.rb]: make it

```ruby
zap delete: [
              REST OF THE STUFF HERE
            ]
```

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 8c4f139609b0618d5d497ee040e3901e5cc56b99. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by Igotit** [comment]
Thanks for the modifications and having Qingg included!

**Pull request closed without merging**